### PR TITLE
[13.0][ADD] mail_activity_permission

### PR DESCRIPTION
**Pull request by hbrunn**
This module allows to manage exceptional access to records via activities.

It is meant for situations where users need access to some record that according to the permissions shouldn't be accessible, but it's also not feasible to change the global permission model.

### CODE REVIEW

**Comment by hbrunn** [comment]
test are fixed in https://github.com/OCA/social/pull/1011

**Review by hieulucky111 — APPROVED** [review]
Code Review
LGTM

File: mail_activity_permission/models/mail_activity_mixin.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        comodel_name="mail.activity",
+        compute="_compute_activity_permission_ids",
+        search="_search_activity_permission_ids",
+        groups="base.group_user",
+        context={"active_test": False},
+    )
+
+    @api.depends("activity_ids")
+    def _compute_activity_permission_ids(self):
+        MailActivity = self.env["mail.activity"].with_context(active_test=False)
```
Comment (CoraVanga) [inline mail_activity_permission/models/mail_activity_mixin.py:22]: I think you should use another variable name, `MailActivity` is duplicated with the class name

**Review by CoraVanga — CHANGES_REQUESTED** [review]

File: mail_activity_permission/models/mail_activity_mixin.py (hunk 1 above)
Comment (hbrunn) [reply mail_activity_permission/models/mail_activity_mixin.py:22]: thanks, I've renamed the class instead to its canonical name

**Review by CoraVanga — APPROVED** [review]
LGTM

File: mail_activity_permission/models/mail_activity_mixin.py (hunk 1 above)
Comment (CoraVanga) [reply mail_activity_permission/models/mail_activity_mixin.py:22]: Thanks, it is ok for me

**Pull request closed without merging**